lib/allscripts_api/magic_params.rb

```diff
@@ -2,7 +2,7 @@
 
 module AllscriptsApi
   # A simple formatter for Magic Action's numbered perameters
-  class NumberedParams
+  class MagicParams
     def self.format(user_id: "", patient_id: "", parameter1: "", parameter2: "", parameter3: "", parameter4: "", parameter5: "", parameter6: "", data: {})
```

Comment:
Thoughts between parameter_1 vs parameter1 or param_1 vs param1

Reply:
no preference. maybe param1, since it's shorter? I basically preserved the Allscripts name for clarity.

Reply:
perhaps we can address this later